src/main/java/tds/exam/results/mappers/ExamineeMapper.java

```diff
@@ -24,12 +24,16 @@
     private static final String DOB_ATTRIBUTE_ID = "DOB";
     private static final Logger log = LoggerFactory.getLogger(ExamineeMapper.class);
 
-    private static final Map<String, String> attributeNames = ImmutableMap.of(
-        DOB_ATTRIBUTE_ID, "Birthday",
-        "Gender", "Sex",
-        "LastName", "LastOrSurname",
-        "SSID", "StudentIdentifier"
-    );
+    private static final Map<String, String> trtAttributeRelationshipTranslations = ImmutableMap.<String, String>builder()
+        .put(DOB_ATTRIBUTE_ID, "Birthday")
+        .put("Gender", "Sex")
+        .put("LastName", "LastOrSurname")
+        .put("SSID", "StudentIdentifier")
+        .put("DistrictID", "ResponsibleDistrictIdentifier")
```

Comment:
The immediate need was to movie the existing code over.  Since it will work with RDW as-is, we should leave it and then we can potentially have it updated later as a Phase 2 thing if necessary.